app/views/tour_list/recordings/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<section class="recordings">
```

Comment:
What is this view for? I'm confused. Is this entire view copy/pasted from somewhere else?

Reply:
This view shows a list of all the recordings for that landmark. The user has the option to pick from this list. It's pretty much the same code from LandmarksShowFacade so I decided to use that facade. We can create a new facade or partial.

This is for user can select existing recording for their tour functionality.